docs/content/components/layout/grid/grid-bento.demo.tsx

```diff
@@ -0,0 +1,74 @@
+import { Grid } from '@marigold/components';
+import { cn } from '@marigold/system';
+
+const Teaser = ({
+  className,
+  src,
+  alt,
+}: {
+  className?: string;
+  src: string;
+  alt: string;
+}) => (
+  <img
+    className={cn(
+      'size-full rounded-2xl object-cover object-center drop-shadow-lg',
+      className
+    )}
+    alt={alt}
+    src={src}
+  />
+);
+
+export default () => (
+  <Grid
+    areas={[
```

Comment:
Maybe I would write some more context how areas, colums and rows props influencing each other?

Reply:
but this would mean copy/pasting mdn? :D

Reply:
Maybe you can write out the important things. I'm just not sure if anyone understands it and people are lazy to search for more infos. But it's just a proposal, not a mistake.

Reply:
added a article to the documentation